assignments/callbacks.js

```diff
@@ -27,24 +27,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length); 
 }
 
+getLength(items, function(tools) {
+  console.log(tools); 
+}) 
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  let a = arr.length -1 
+  return cb(arr[a]); 
 }
+last(items, function(tripEssentials){
+  console.log(tripEssentials); 
+}) 
+
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
+sumNums(5, 10, function(numbers){
+  console.log(numbers); 
+})
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y)
 }
+multiplyNums(9, 4, function(multiplies){
+  console.log(multiplies); 
+})
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i = 0; i < list.length; i++ ){
+    if (list[i] === item) {
+      return cb(true);
+    }
+  }
+  return cb(false); 
 }
+contains("soccer", items, function(flag){
+  console.log(flag); 
+})
+
+contains("soccer", items, function(flag){
+  console,log(flag); 
```

Comment:
console error `log is not defined`

There seems to be a syntax error in this line